profiler/src/main/java/com/splunk/opentelemetry/profiler/JfrRecorder.java

```diff
@@ -63,7 +66,8 @@ Recording newRecording() {
 
   public void flushSnapshot() {
     try (Recording snap = jfr.takeSnapshot()) {
-      Path path = Path.of(Instant.now().toString() + ".jfr");
+      Path file = Paths.get(Instant.now().toString() + ".jfr");
```

Comment:
What does `Instant.now().toString()` look like, anyway? Feels like it would be better to not use a `toString()` method for generating filenames, since there's no guarantee that it will generate usable filenames on misc. filesystems. For instance, what if the `toString()` changes to look like `06/03/2021`? Seems like that would be bad.

Reply:
Could use `File.createTempFile`

Reply:
That's a good call.  The name isn't that important, but it's nice to have them be sortable and also human readable.  Lemme find something better.